From 2e1fbe466f798841db5115a2e6915888d02336bd Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Fran=C3=A7ois=20Revol?= <revol@free.fr>
Date: Mon, 27 Oct 2014 22:19:34 +0100
Subject: [PATCH] configure.ac: Do the AM_CONDITIONAL() after finally setting
 the variable

Avoids trying to compile qphotorec when Qt is not available.
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 4d54612..016b686 100644
--- a/configure.ac
+++ b/configure.ac
@@ -311,7 +311,6 @@ case "$target" in
     ;;
 esac
 
-AM_CONDITIONAL(USEQT, test "$use_qt" = true)
 AM_CONDITIONAL(USEICON, test "$use_icon" = yes)
 
 # Checks for programs.
@@ -854,6 +853,7 @@ then
   qphotorec_LDADD="$qphotorec_LDADD $QTGUI_LIBS"
   qphotorec_CXXFLAGS="$qphotorec_CXXFLAGS $QTGUI_CFLAGS"
 fi
+AM_CONDITIONAL(USEQT, test "$use_qt" = true)
 
 # e2fsprogs may be using pthread
 # checks for pthreads
-- 
1.8.3.4

